packages/discovery-provider/ddl/migrations/0107_add_email_tables.sql

```diff
@@ -0,0 +1,46 @@
+begin;
+
+-- Store encrypted emails with their metadata
+CREATE TABLE IF NOT EXISTS encrypted_emails (
+    email_id INTEGER PRIMARY KEY,
+    seller_user_id INTEGER NOT NULL,
+    encrypted_email TEXT NOT NULL  -- base64 encoded
+);
+
+COMMENT ON TABLE encrypted_emails IS 'Stores encrypted emails and their metadata for secure communication between sellers and buyers';
+COMMENT ON COLUMN encrypted_emails.email_id IS 'Unique identifier for each encrypted email record';
+COMMENT ON COLUMN encrypted_emails.seller_user_id IS 'User ID of the seller who owns the encrypted email';
+COMMENT ON COLUMN encrypted_emails.encrypted_email IS 'Base64 encoded encrypted email content';
+
+-- Store the encryption keys for sellers and grantees
```

Comment:
Why two different tables?

Reply:
i made it two tables so we can clearly define ownership. 

say rickyrombo manages romboricky but is also an aritst, the separate tables make it a bit more clear to handle revocation of romboricky's emails from rickyrombo since he is just a grantee, but rickyrombo maintains the emails he owns as an artist in the `email_encryption_keys` table. we could join these into a single table, but there will be a lot of null values since not all users have grantees, wdyt?

Reply:
Yeah makes sense. The owner key here is a shared secret still from the owner to the owner right?

Let's update the comment though because grantees aren't in this table